packs/vsphere/actions/vm_hw_details_get.py

```diff
@@ -0,0 +1,12 @@
+from vmwarelib import inventory
+from vmwarelib.actions import BaseAction
+
+
+class GetVMDetails(BaseAction):
+    def run(self, vms):
+        results = []
+        for vm in vms:
+            vm = inventory.get_virtualmachine(self.si_content, name=vm)
```

Comment:
This is ok for now but using a propertyspec to retrieve all at the same time will be better. Please leave a TODO for future updates.